### DISCUSSION TITLE
Provide descriptive Text to the Buttons/Footer on web ui

### DISCUSSION THREAD

**Discussion by turbolooser** [revk/xESP32-Faikout#195] [General]
I love your Faikin Project and am wondering if it would be possible to add some descriptive Textboxes to these Icon Only switches. I do not feel like these icons are self explaining :-) Maybe its only me. Could be good enough to add a footer which explains or just add some documentation here on the repo

![image](https://github.com/revk/ESP32-Faikin/assets/115738363/b5a363c3-52ef-4052-9391-be6a41e64e70)

**Comment by revk** [comment]
The buttons have a tool tip - does that not help?

<img width="240" alt="Screenshot 2024-01-23 at 10 36 54" src="https://github.com/revk/ESP32-Faikin/assets/996983/e16f4303-c795-4d54-ac03-a4cb042ae88b">

**Reply by turbolooser** [reply]
Ohh my bad... you are right :-)
I was just not waiting long enough for those to popup ... thats perfect thank you

**Reply by njh** [reply]
Tool tips work on Desktop browsers but I am not sure how to get them to display on mobile (iOS for me)?

I was using Faikin on my phone the other night, wondering what all the new settings / icons were for!
